ci/requirements-conversion.txt

```diff
@@ -22,10 +22,10 @@ decorator==4.4.2
     # via networkx
 defusedxml==0.7.1
     # via
-    #   -r ${INTEL_OPENVINO_DIR}/deployment_tools/model_optimizer/requirements_caffe.txt
-    #   -r ${INTEL_OPENVINO_DIR}/deployment_tools/model_optimizer/requirements_mxnet.txt
-    #   -r ${INTEL_OPENVINO_DIR}/deployment_tools/model_optimizer/requirements_onnx.txt
-    #   -r ${INTEL_OPENVINO_DIR}/deployment_tools/model_optimizer/requirements_tf2.txt
+    #   -r <openvino repository>/tools/mo/requirements_caffe.txt
+    #   -r <openvino repository>/tools/mo/requirements_mxnet.txt
+    #   -r <openvino repository>/tools/mo/requirements_onnx.txt
+    #   -r <openvino repository>/tools/mo/requirements_tf2.txt
```

Comment:
The file is generated automatically by https://github.com/openvinotoolkit/open_model_zoo/blob/master/ci/update-requirements.py. You need to update the script

Reply:
these locations are nor valid anyway. MO is not part of OpenVINO Installer, it's installed only via PyPi with dependencies specified using extra notation

Reply:
These ci/requirements* are only used in our CI. They aren't redistributed. If you want to update them, you should target 21.2. As far as I remember, 21.2 minimal python version is going to be 3.8. If ov has already moved to python3.8, you can update the requirements, if not, we shouldn't bother for now.